### ISSUE TITLE
Error using React Markdown editor

### ISSUE THREAD

**Issue by johnsonsamuel** [JedWatson/react-md-editor#18]
Hello I am trying to implement react markdown editor and I have used Jed Watson's http://jedwatson.github.io/react-md-editor/

But the problem is that, I am getting the error attached. Any thoughts what I am missing :(.. Has anyone encountered similar issue ?

import Editor from 'react-md-editor';

state = {
              code: '# React Markdown '            

}

updateCode = (newCode) => {
      this.setState({
          code: newCode
      });
    }

Render:

<Editor value={this.state.code} onChange={this.updateCode} />
<img width="1252" alt="screen shot 2018-05-28 at 1 33 55 pm" src="https://user-images.githubusercontent.com/4627394/40604364-7263c122-627c-11e8-9d32-0cbebf4c16bc.png">

**Comment by jordan-enev** [comment]
TL;DR - Project's PropTypes usage is incompatible with the project's React version.

Here's the detailed answer: https://stackoverflow.com/a/50561962/4312466
